lib/pbench/server/auth/__init__.py

```diff
@@ -123,24 +126,30 @@ def get(
         Returns:
             Response from the request.
         """
-        return self._method("GET", path, None, headers=headers, **kwargs)
+        return self._method("GET", path, None, None, headers=headers, **kwargs)
 
     def post(
-        self, path: str, data: Dict, headers: Optional[Dict] = None, **kwargs
+        self,
+        path: str,
+        data: Union[Dict, str] = None,
+        json: Optional[Dict] = None,
+        headers: Optional[Dict] = None,
+        **kwargs,
     ) -> requests.Response:
         """POST wrapper to handle an authenticated POST operation on the
         Resource at a given path.
 
         Args:
             path : Path for the request
-            data : JSON request body
+            data : Request body to attach
+            json: JSON request body
```

Comment:
There should be a space before the colon in line 145.

Reply:
I don't actually like the extra space: but one way or the other we should be consistent at least within the `Args` list for a function, and preferably at least through the module ...

Reply:
I don't like the extra space either, but it seems to be the established convention and so I think we should try to conform to it (and, as Dave says, the lines should definitely match the others next to them...).